openshift/init-cluster.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+PATH_TO_ANSICLOUD=/home/mveber/projekty/ansicloud-full/ansible-2.9
+PATH_TO_PULL_SECRETS=$PWD/pull-secrets.json
+
+
+kind create cluster
+helm install cert-manager jetstack/cert-manager --namespace cert-manager --create-namespace --set crds.enabled=true
+
+. $PATH_TO_ANSICLOUD/bin/activate
+export AWS_REGION="us-west-2"
+export AWS_ACCOUNT_ID=471112697682
```

Comment:
remove account-id value, set any fake num

Reply:
I added there a condition checking if AWS_ACCOUNT_ID is defined externaly, ok?